test/src/androidTest/java/co/condorlabs/customcomponents/test/BaseEditSaveStateTest.kt

```diff
@@ -0,0 +1,69 @@
+package co.condorlabs.customcomponents.test
+
+import android.view.View
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.filters.SmallTest
+import androidx.test.runner.AndroidJUnit4
+import co.condorlabs.customcomponents.custombutton.CustomButton
+import co.condorlabs.customcomponents.customedittext.*
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Created by Oscar Tigreros on 25,July,2019
+ */
+@RunWith(AndroidJUnit4::class)
+class BaseEditSaveStateTest : MockActivityTest2() {
```

Comment:
Why is MockActivityTest2 different?

Reply:
because use a different activity to start

Reply:
I think you should express that difference in the name.